### PR TITLE
Added Pulumi code to deploy lambda with powertools

### PR DESCRIPTION
**Pull request by pierskarsenbarg**
## Description of your changes

Added code snippet to show how to deploy lambda with the powertools layer using Pulumi

### How to verify this change

image

### Related issues, RFCs

**Issue number:** 

### PR status

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x] I have performed a *self-review* of my own code
- [ ] I have *commented* my code where necessary, particularly in areas that should be flagged with a TODO, or hard-to-understand areas
- [x] I have made corresponding changes to the *documentation*
- [ ] I have made corresponding changes to the *examples*
- [ ] My changes generate *no new warnings*
- [ ] The *code coverage* hasn't decreased
- [ ] I have *added tests* that prove my change is effective and works
- [ ] New and existing *unit tests pass* locally and in Github Actions
- [ ] Any *dependent changes have been merged and published* in downstream module
- [ ] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

### Breaking change checklist

- [ ] I have documented the migration process
- [ ] I have added, implemented necessary warnings (if it can live side by side)

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -186,6 +186,33 @@ You can include Lambda Powertools Lambda Layer using [AWS Lambda Console](https:
...
+            ],
+            code: new pulumi.asset.AssetArchive({
+                ".": new pulumi.asset.FileArchive("./app")
+            }),
+            tracingConfig: {
+                mode: "Active"
+            },
+            runtime: aws.lambda.Runtime.NodeJS16dX,
+            handler: "index.handler",
+            role: role.arn
```
Comment (leandrodamascena) [inline docs/index.md:212]:
```suggestion
            role: role.arn,
            architectures: ["x86_64"]
```

Comment (pierskarsenbarg) [reply docs/index.md:210]: This is the default value for the architectures input, so you don't _have_ to have it there. Is it a best practice to include it?

Comment (leandrodamascena) [reply docs/index.md:210]: Yes indeed `x86_64` is the default value if not informed. it was just a suggestion to keep this option more explicit.

Comment (pierskarsenbarg) [reply docs/index.md]: Sorry, I meant that I think we set it by default behind the scenes if it's not set (presumably you do as well)

**Review by dreamorosi — APPROVED** [review]

**Comment by dreamorosi** [comment]
Thank you @pierskarsenbarg for contributing this sample!

and thank you @leandrodamascena for the review.

**Pull request merged**